string siftModel
string colorModel
string trackingMode
float32 maxPositionDifference
float32 maxColorDifference
float32 maxSizeDifference
float32 positionFactor
float32 colorFactor
float32 sizeFactor
float32 maxTotalDifference
int32 upperSizeLimit
int32 lowerSizeLimit
int32 minPixelCount
string selectionMode
float32 colorDistanceThreshold
float32 siftScales
float32 siftInitSigma
float32 siftPeakThreshold
float32 flannKnn
float32 flannMatchFactor
float32 flannMatchCount
